### ISSUE TITLE
Computed property volatility deprecated in Ember 3.8 / update keycloak-adapter

### ISSUE THREAD

**Issue by MikeEdgar** [JFTechnology/ember-keycloak-auth#18]
Update addon/mixins/keycloak-adapter.js to use native getter.

Replace the following function:
```javascript
headers: computed(function() {

    let session = this.get('session');
    let keycloak = session.get('keycloak');

    return {
      'Authorization': `Bearer ${keycloak['token']}`,
    };

}).volatile()
```

With native getter for `headers`:
```javascript
get headers() {
    let session = this.get('session');
    let keycloak = session.get('keycloak');
    return {
      'Authorization': `Bearer ${keycloak['token']}`
    };
}
```

**Comment by MikeEdgar** [comment]
Link to Ember deprecation notice: https://deprecations.emberjs.com/v3.x/#toc_computed-property-volatile

**Comment by MikeEdgar** [comment]
Fixed by #19
